person.rb

```diff
@@ -1,3 +1,8 @@
+require './nameable'
+require './decorator'
+require './capitalize_decorator'
+require './trimmer_decorator'
+
 class Person
```

Comment:
Great work so far 👍 according to the project [requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_decorator_v2.md), It would be better if you inherit `Nameable` class. Please Make sure that your `Person` class inherits from `Nameable`.🥱

Reply:
Hi @AbdusSattar-70 thank you for the review.
Changes done: Person class updated to inherit from Nameable.